performance/graphite-setup.yaml

```diff
@@ -0,0 +1,102 @@
+dse_version: 6.0.11
+# Driver branch to use
+driver_oss_branch: 4.x
+# Driver dse branch to use
+driver_dse_branch: 4.x
+# Driver version identifier (used as part of graphite prefix)
+driver_version: 4.8.0
+# Driver examples branch to use
+driver_examples_branch: java-driver-4.x
+# How long to run test for
+duration: 2d
+
+---
+
+ensemble:
+  observer:
+    node.count: 1
+    provisioner:
+      name: ctool
+      properties:
+        kill_cluster_after_test: false
+        cloud.provider: openstack
+        cloud.tenant: performance
+        cloud.instance.type: ms1.small
+    configuration_manager:
+      - name: ctool_monitoring
+        properties:
+          graphite.create_server: true
+  server:
+    node.count: 3
+    provisioner:
+      name: ctool
+      properties:
+        kill_cluster_after_test: false
```

Comment:
Isn't this replaced with `reuse`?

Reply:
what do you mean? This property is still valid

Reply:
It's deprecated: https://fallout.sjc.dsinternal.org/components/provisioners/CTool#fallout.provisioner.ctool.kill_cluster_after_test